### PR TITLE
Configurable branch and status file path.

### PR DESCRIPTION
**Pull request by glogiotatidis**

### CODE REVIEW

**Comment by glogiotatidis** [comment]
@jgmize r?

File: app/worker.py [hunk 1]
```diff
@@ -21,15 +21,15 @@ def handler(event, context):
             raise Exception('Repository {}/{} does not exist'.format(config.GITHUB_ORG,
                                                                      config.GITHUB_REPOSITORY))
 
-        status_file = repository.file_contents(config.STATUS_FILE, ref='gh-pages')
+        status_file = repository.file_contents(config.STATUS_FILE, ref=config.BRANCH)
```
Comment (johngian) [inline app/worker.py:24]: Default value here is going to be `master` instead of `gh-pages`. Is that expected?

Comment (glogiotatidis) [reply app/worker.py:24]: Yes we're going to default to master to keep pages and worker code in one branch. See also https://github.com/blog/2228-simpler-github-pages-publishing

**Comment by johngian** [comment]
r+

**Comment by glogiotatidis** [comment]
Thanks!

**Pull request merged**